player/lua/console.lua

```diff
@@ -609,6 +611,15 @@ function get_clipboard(clip)
         if not res.error then
             return res.stdout
         end
+    elseif platform == 'wayland' then
+        local res = utils.subprocess({
+            args = clip and { 'wl-paste', '--no-newline' }
+                or { 'wl-paste', '--primary', '--no-newline' },
```

Comment:
You could use the short options for wl-paste here instead and fit this on one line.